usaspending_api/etl/management/commands/load_multiple_submissions.py

```diff
@@ -26,7 +25,7 @@ def handle(self, *args, **options):
             broker_cursor = broker_conn.cursor()
             api_conn = connections["default"]
             api_cursor = api_conn.cursor()
-        except Exception as err:  # NOQA
+        except Exception as err:
```

Comment:
Strange that this causes code climate issues for me but not for thee

Reply:
I don't understand why it would be a CC issue. `err` is used below

Since CC checks only run for dev PRs, this _could_ still be flagged when it reaches that env.

Reply:
CC gave errors on `except Exception` for being too generic, not on the variable

Reply:
While that is a code smell, CC shouldn't report that as an error.

Reply:
I guess this is a fight for you and CC to have when you try and push this to dev